docs/introduction/agents.md

```diff
@@ -1,24 +1,25 @@
-# Agent
+# Agents
 
-The agent vocabulary describes networks of people and groups, using agents and their relationships.
+The agent vocabulary describes networks of people, groups, and organizations, constructed using a simple but powerful model of agents and their relationships.
```

Comment:
This isn't very important, but I'd prefer:
>The agent vocabulary describes networks of people, organizations and networks

Reasons:
1.  To me, "group" is an extremely generic term including all organizations & networks, and isn't needed here.
2.  I perceive "organizations" to mean "structured groups" or "formally structured groups", including all "agentified groups" or "group agents" as well as many networks.   
3.  I perceive "networks" to broadly mean all sets of persistently interactive people (who can send and/or receive info, energy or matter to each other.)  Networks includes many huge groups of peers with no internal group structure.  To me, this seems to be both properly generic *and* distinctly contrasted with--although sometimes overlapping-- "organization".

Some people might dislike the intentional fractal redundancy created by explicitly referring to "networks of networks".  I personally think it's appropriate here.